src/interactive.js

```diff
@@ -0,0 +1,39 @@
+import Methods from './utils.js';
+
+export default class Interactive {
+static toggleCompleted = (id, curStatus) => {
+  const todoList = Methods.getLocalStorageData();
+  todoList[id].completed = curStatus;
+  Methods.setLocalStorageData(todoList);
+  Methods.showTodoItems();
+}
+
+static addCheckEvent = () => {
+  document.querySelectorAll('.checkbox').forEach((box) => box.addEventListener('change', () => {
```

Comment:
For the JS best practices, Try to avoid the line that has more than 80 characters, because it gets harder to follow, instead you can use : 
``` 
document.querySelectorAll('.checkbox')
        .forEach( box => box.addEventListener('change'), () => {
        ......}
```